precode.sh

```diff
@@ -7,37 +7,56 @@
 #   dir3
 #       dir4
 
+mkdir task
```

Comment:
Можно сделать одной командой
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```